pkg/server/errorshandler/handler.go

```diff
@@ -48,6 +52,10 @@ func (handler *Handler) process(body []byte) ResponseMessage {
 		return ResponseMessage{true, fmt.Sprintf("%s", err)}
 	}
 
+	if handler.RedisClient.IsBlocked(projectId) {
+		return ResponseMessage{true, "Project has exceeded the event limit"}
```

Comment:
```suggestion
		return ResponseMessage{true, "Project has exceeded the events limit"}
```
может правильнее так?